envs/eks/storage-class.yaml

```diff
@@ -16,4 +16,4 @@ parameters:
   encrypted: 'true'
   type: gp2
 volumeBindingMode: WaitForFirstConsumer
-reclaimPolicy: Retain  # Useful in case you delete the PersistentVolumeClaim
+reclaimPolicy: Delete  # Change to retain, useful in case you delete the PersistentVolumeClaim
```

Comment:
The deployment of this repository is aimed at environments that can be productive, so it is necessary to have the retain option by default